### PR TITLE
Cleanup .gitignore file

### PR DESCRIPTION
**Pull request by shulard**
I am targetting this branch, because this is a pedantic branch.

Related to: sonata-project/dev-kit#211
## Subject

Remove plateform specific ignore entries from .gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-
-# composer related
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
-composer.phar
-vendor
+phpunit.xml
+.php_cs.cache
```
Comment (core23) [inline .gitignore]: Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore

Comment (core23) [inline .gitignore]: Ping @shulard

**Comment by core23** [comment]
Please rebase, this will retrigger StyleCI. After that we will merge this. @shulard

**Comment by shulard** [comment]
The build have been triggered here. Seems that the master branch fails on Travis too... Do you want to wait master for a green rebase ?

**Comment by greg0ire** [comment]
Hmmm… no. IIRC, we're far from a green build on this bundle.

**Pull request merged**